src/outbox/outboxActions.js

```diff
@@ -15,7 +15,7 @@ import { sendMessage } from '../api';
 import {
   getSelfUserDetail,
   getUserByEmail,
-  getAllUserAutocompleteOptions,
+  getAllMentionsSuggestions,
```

Comment:
Still not a good name :)

Reply:
We use "Wildcard" as the word to refer to `@all` / `@everyone` mentions elsewhere in the codebase.

Reply:
Good suggestion.

@vishwesh3 I think the naming should make it obvious that you are getting all users, but also wildcards. We need to think of scenarios: when do we want only users vs users and wildcards?

The simplest naming I guess would be `getUsersAndWildcards`.
There will be another function, that I might work on next, that will get all users but grouped by recent and first letter. This will be another `get users` function, that will be used in the UserList instead of what we do now.

Reply:
Done 👍